atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/ConsecutiveNarrowTable.java

```diff
@@ -103,6 +105,26 @@ protected void setupData() {
     }
 
     @State(Scope.Benchmark)
+    public static class RegeneratingCleanNarrowTable extends CleanNarrowTable {
+        @TearDown(Level.Invocation)
```

Comment:
Confirmed benchmarks do take long enough at least on my machine that I think this is okay.